portfolio/src/main/webapp/script.js

```diff
@@ -494,49 +494,7 @@ function route() {
   directionsService.route(request, function(result, status) {
     if (status === 'OK') {
       console.log("Number of alternative routes: " + result.routes.length);
-
-      var counter = 0;
-      var routeIndex = 0;
-      var lessCrimesInRoute = 1000;
-
-      for (var r = 0; r < result.routes.length; r++) {
-        var route = result.routes[r];
-
-        for (var j = 0; j < route.legs[0].steps.length; j++) {
-          routeArray = new google.maps.Polyline({
-            path: [
-              new google.maps.LatLng(route.legs[0].steps[j].start_location.lat(), route.legs[0].steps[j].start_location.lng()),
-              new google.maps.LatLng(route.legs[0].steps[j].end_location.lat(), route.legs[0].steps[j].end_location.lng())
-            ]
-          });
-          fetch('/markers').then(response => response.json()).then((markers) => {
-            markers.forEach((marker) => {
-              var myPosition = new google.maps.LatLng(marker.lat, marker.lng);
-              if (google.maps.geometry.poly.isLocationOnEdge(myPosition, routeArray, 0.0064)) {
-                counter += rateCrime(marker.crimeType);
-                console.log("Rating" + counter);
-                console.log(marker.lat);
-                console.log(marker.lng);
-              }
-            });
-          });
-        }
-        console.log("Rating" + counter);
-        if (counter < lessCrimesInRoute) {
-          lessCrimesInRoute = counter;
-          routeIndex = r;
-        }
-        counter = 0;
-      }
-
-      console.log("Chosen route index: " + routeIndex);
-      /*directionsDisplay.setDirections(result);*/
-
-      new google.maps.DirectionsRenderer({
-        map: map,
-        directions: result,
-        routeIndex: routeIndex
-      });
+      pickSafestRoute(result);
```

Comment:
revert the changes in this file since salvador should be committing this part (unless I'm mistaken and he won't have this change)

Reply:
Okay, I'll put the original part there then, thank you!